.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: latest next podman-next ${{ github.sha }}
+  IMAGE_REGISTRY: quay.io/podman
+  COPR_OWNER: rhcontainerbot
+  COPR_PROJECT: podman-next
+
+jobs:
+  fcos-podman-next-image-build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+    - name: Wait for successful podman-next build with the latest commit
+      run: |
+        # TODO: add this in the Containerfile itself or as a --build-arg
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman $(git rev-parse --short ${{ github.sha }})
+        echo "podman-next build successful."
+
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Build FCOS Image
+      id: build_image
+      # Ref: https://github.com/redhat-actions/buildah-build
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
+        tags: ${{env.IMAGE_TAGS }}
+        containerfiles: |
+          ./contrib/podman-next/fcos-podmanimage/Containerfile .
+
+    - name: Push to Quay
+      id: push-to-quay
+      # Ref: https://github.com/redhat-actions/push-to-registry
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: ${{ steps.build-image.outputs.image }}
+        tags: ${{ steps.build-image.outputs.tags }}
+        registry: ${{ env.IMAGE_REGISTRY }}
+        username: ${{ secrets.QUAY_PODMAN_USERNAME }}
+        password: ${{ secrets.QUAY_PODMAN_PASSWORD }}
```

Comment:
Just as a reminder: It would be really great to have some kind of notification if the job fails.  Basically nobody pays any attention to the 'Actions' tab of the repo. and it seems important to know if the build fails for some reason.

Reply:
remind me, do you know of a failure notification mechanism in GHA?